### ISSUE TITLE
Add alarm variables to the response of chart and data

### ISSUE THREAD

**Issue by cakrit** [netdata/netdata#6054]
`api/v1/info/alarm_variables?chart=XX` returns the variables available for creation of alarms.

We have a request to add them to both `/chart` and `/data` call responses as well.

_Originally posted by @ilyam8 in https://github.com/netdata/netdata/issues/5990#issuecomment-491498124_

**Comment by alpes214** [comment]
@ilyam8  @cakrit 
The /alarm_variables?chart=XX request has chart_variables field in its response. 
- same field with the same name and values should be added to the response for /chart?chart=XX request.
- the /data request has many different response formats, some of them like csv cannot be extended easily. Therefor chart_variables should be added to response when /data request is called with options=jsonwrap parameter only: /data?chart=XX&options=jsonwrap

I doubt about /data request.
@ilyam8 is my understanding correct?

**Comment by ilyam8** [comment]
> same field with the same name and values should be added to the response for /chart?chart=XX request.

Not sure.

I have

```python
CHARTS = {
    'random': {
        'options': [None, 'A random number', 'random number', 'random', 'random', 'line'],
        'lines': [
            ['random1']
        ],
        'variables': [
            ['var_1', 999],
            ['var_2', 999],
            ['var_3', 999]
        ]
    }
}
```

There you can see 3 chart variables.

I did `/alarm_variables?chart=example.random`. My `chart_variables`:

```json
{
    "var_1": 999,
    "var_3": 999,
    "var_2": 999,
    "random2_raw": 38,
    "red": null,
    "update_every": 1,
    "random3_raw": 49,
    "random1_raw": 31,
    "random1_last_collected_t": 1565088357.00000,
    "random2_last_collected_t": 1565088357.00000,
    "last_collected_t": 1565088357.00000,
    "random1": 0,
    "random2": 0,
    "random3": 0,
    "green": null,
    "random3_last_collected_t": 1565088357.00000,
    "collected_total_raw": 118
}
```

i think we need only `var_1`, `var_2` and `var_3` from there.

**Comment by ilyam8** [comment]
Maybe it is not even possible to distinguish between user defined chart vars and internally created, it looks like it is our case :sweat_smile: If so then yes

> same field with the same name and values should be added to the response for /chart?chart=XX request.

**Comment by ilyam8** [comment]
> Maybe it is not even possible to distinguish between user defined chart vars and internally created

It is possible

> @vlvkobal The RRDVAR_OPTION_CUSTOM_CHART_VAR option is set for custom chart variables.

**Comment by alpes214** [comment]
> It is possible

I will use RRDVAR_OPTION_CUSTOM_CHART_VAR to put only custom variables into response
Thank you @ilyam8

**Comment by ilyam8** [comment]
fixed in #6615 	

thx @alpes214 !

**Comment by ilyam8** [comment]
@alpes214 looks like adding issue N to every commit msg is not a good idea, it doesnt add much value, see all those references above^^
